client/src/api/index.js

```diff
@@ -126,11 +126,16 @@ export default {
       }),
     requests: {
       filter: (id, query = {}, authorization) =>
-        request(getAPIURL(`materials/${id}/requests/`, query), {
+        request(getAPIURL(`materials/${id}/requests`, query), {
```

Comment:
Trailing / on this one is not neccessary?

Reply:
it actually doesn't work at the moment